scoring.js

```diff
@@ -0,0 +1,43 @@
+const calculateScore = (player) => {
+
+  switch (player.position) {
+    case 'QB':
+      return passingScore() + rushingScore()
+    case 'RB':
+    case 'WR':
+      return rushingScore() + kickReturnScore() + puntReturnScore() + receivingScore()
+    case 'TE':
+      return receivingScore()
+    default:
+      return 0
+  }
+
+
+  function passingScore() {
```

Comment:
`calculatePassingScore` would be a better name. Functions do things so their names should involve be active verbs. `passingScore` sounds like a variable that holds the passing score.